suitcase/tiff/tests.py

```diff
@@ -1,2 +1,67 @@
-# Tests should generate (and then clean up) any files they need for testing. No
-# binary files should be included in the repository.
+from . import export
+import numpy
+from numpy.testing import assert_array_equal
+import pytest
+from suitcase.utils.conftest import (simple_plan,
+                                     multi_stream_one_descriptor_plan,
+                                     one_stream_multi_descriptors_plan)
+import tempfile
+import tifffile
+
+@pytest.fixture()
+def expected():
```

Comment:
This can just be a global constant. No need to generate a fresh copy each time, because it is not consumed or mutated.

Reply:
done